### PR TITLE
reformat tabs shortcode

### PR DESCRIPTION
**Pull request by kbhawkey**
Reformatted tabs shortcode to fix rendered HTML.
Tested with Hugo 0.69.2
Note: The `note` shortcode embedded in lists is not rendering correctly (adjusted for this).

### CODE REVIEW

**Comment by kbhawkey** [comment]
I can remove the netlify and config toml changes from this PR (opening this up to move forward with the Hugo upgrade).

**Comment by tengqm** [comment]
@jimangel

**Comment by jimangel** [comment]
This looks great @kbhawkey! If it looks the same for `.59` my vote is to update the commits and let's merge this!

**Comment by kbhawkey** [comment]
> 
> 
> This looks great @kbhawkey! If it looks the same for `.59` my vote is to update the commits and let's merge this!

Hi! I think it makes sense to use a version of Hugo that does not have this issue (for example, <= 0.68.x) or incorporate a workaround until the issue is fixed.
I can test this change with another version of Hugo.
I'd like to use this PR to continue to create 0.69.x docs changes.

**Comment by kbhawkey** [comment]
> 
> 
> This looks great @kbhawkey! If it looks the same for `.59` my vote is to update the commits and let's merge this!

Hi! I think it makes sense to use a version of Hugo that does not have this issue (for example, <= 0.68.x) or incorporate a workaround until the issue is fixed.
I can test this change with another version of Hugo.
I'd like to use this PR to continue to create 0.69.x docs changes.

**Comment by kbhawkey** [comment]
> 
> 
> This looks great @kbhawkey! If it looks the same for `.59` my vote is to update the commits and let's merge this!

Hi! I think it makes sense to use a version of Hugo that does not have this issue (for example, <= 0.68.x) or incorporate a workaround until the issue is fixed.
I can test this change with another version of Hugo.
I'd like to use this PR to continue to create 0.69.x docs changes.

**Comment by kbhawkey** [comment]
> 
> 
> This looks great @kbhawkey! If it looks the same for `.59` my vote is to update the commits and let's merge this!

Hi! I think it makes sense to use a version of Hugo that does not have this issue (for example, <= 0.68.x) or incorporate a workaround until the issue is fixed.
I can test this change with another version of Hugo.
I'd like to use this PR to continue to create 0.69.x docs changes.

**Comment by kbhawkey** [comment]
Page previews:
https://deploy-preview-20245--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-minikube/
https://deploy-preview-20245--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl/
https://deploy-preview-20245--kubernetes-io-master-staging.netlify.app/docs/setup/learning-environment/minikube/

**Comment by kbhawkey** [comment]
@sftim @onlydole, If you have a second, would appreciate more eyes on the `tabs` shortcode changes. Thanks!

File: layouts/shortcodes/tabs.html [hunk 1]
```diff
@@ -2,40 +2,41 @@
...
+{{ else}}
+{{ $path := path.Join $.Page.File.Dir .include }}
+{{ $page := site.GetPage "page" $path }}
+{{ with $page }}
+{{ .Content }}
+{{ else }}
+{{ errorf "[%s] tabs include not found for path %q" site.Language.Lang $path}}
+{{ end }}
+{{ end }}
+{{ end }}
```
Comment (tengqm) [inline layouts/shortcodes/tabs.html:39]: Can you indent the above code a little bit for easier review and maintenance? It is unclear which line has been changed.

Comment (kbhawkey) [reply layouts/shortcodes/tabs.html:39]: I can _retest_ these changes. I tried a number of ways to indent the code and
ended with this format.

Comment (tengqm) [reply layouts/shortcodes/tabs.html:39]: I see ... maybe it is the markdown parser again ...

Comment (kbhawkey) [reply layouts/shortcodes/tabs.html]: Thanks. I cleaned up the template instructions (removing trailing spaces).
The note shortcode has a problem when embedded in lists (blockquote?).

**Comment by kbhawkey** [comment]
Will rebase. Depends upon #19907 .

**Comment by jimangel** [comment]
@kbhawkey this looks good to me! Want to reset the netlify / config files? If it still looks good on the current site we can merge as is and I can rebase https://github.com/kubernetes/website/pull/19907 for review.

**Comment by zacharysarah** [comment]
@kbhawkey #19907 is ready to merge. Is this PR merge-able after rebase? Thanks!

File: layouts/shortcodes/tabs.html [hunk 2]
```diff
@@ -1,43 +1,43 @@
-{{ .Page.Scratch.Add "tabset-counter" 1 }}
-{{ $tab_set_id := .Get "name" | default (printf "tabset-%s-%d" (.Page.RelPermalink) (.Page.Scratch.Get "tabset-counter") ) | anchorize }}
-{{ $tabs := .Scratch.Get "tabs" }}
-{{ if .Inner }}{{/* We don't use the inner content, but Hugo will complain if we don't reference it. */}}{{ end }}
+{{- .Page.Scratch.Add "tabset-counter" 1 -}}
+{{- $tab_set_id := .Get "name" | default (printf "tabset-%s-%d" (.Page.RelPermalink) (.Page.Scratch.Get "tabset-counter") ) | anchorize -}}
+{{- $tabs := .Scratch.Get "tabs" -}}
+{{- if .Inner -}}{{- /* We don't use the inner content, but Hugo will complain if we don't reference it. */ -}}{{- end -}}
```
Comment (zacharysarah) [inline layouts/shortcodes/tabs.html:4]: 💯 🙌 😂

**Comment by zacharysarah** [comment]
Holding `/lgtm` for another reviewer. @jimangel @sftim 

/approve

**Pull request merged**